### DISCUSSION TITLE
not able to upload the video in Upload large video

### DISCUSSION THREAD

**Discussion by srikanth-6520** [Sunbird-Ed/Community#591] [Bugs] [answered]
Hello @amith , @princegupta1131

   we are trying to upload the video for Upload large video in portal , but while uploading it's throwing an error , below we have shared screenshot.
   
![image](https://github.com/Sunbird-Ed/Community/assets/133200106/60459d8b-2cba-457f-805b-6b5cf7c7c27e)

**Comment by Krishnaj20** [comment]
@gandham-santhosh - Can you look into this issue? 

@chitranshu-keshav - fyi pls. This wasn't responded for more than 5 days. 

@amitpriyadarshi @vinukumar-vs

**Comment by amitpriyadarshi** [comment]
@srikanth-6520 as per our previous discussion, it is 403 forbidden error. It means CSP has provided the link but while calling it is throwing authorisation error. It seems some CSP level some configuration error is there while generating this url link. Please cross check with @tsprasath or @gandham-santhosh 

cc: @Krishnaj20 @vinukumar-vs

**Reply by srikanth-6520** [reply]
sure @amitpriyadarshi , i'll try to check with @tsprasath

**Reply by srikanth-6520** [reply]
@tsprasath , i have tried to upload the video through upload content , from upload content we can able to upload the video , but while uploading the same video in upload large video object only giving the above error.

**Reply by Krishnaj20** [reply]
@gandham-santhosh - are you looking into this thread? Pls assist Srikanth from your team.

**Reply by gandham-santhosh** [reply]
Yes, @Krishnaj20. @srikanth-6520  will update the thread with resolution steps and close once it is resolved

**Reply by Krishnaj20** [reply]
@gandham-santhosh - Is this issue still open?

**Comment by amitpriyadarshi** [comment]
@srikanth-6520 as per the [comment](https://github.com/orgs/Sunbird-Ed/discussions/591#discussioncomment-6593250) this is AWS url configuration issue. As this configuration can be validated and fixed internally, marking this as Closed.

cc: @Krishnaj20 @vinukumar-vs

**Reply by srikanth-6520** [reply]
@amitpriyadarshi , yesterday we have checked with @rajsimman , as per his analysis , issue not causing with the AWS configurations , as per his investigation , some of the plugins are causing this issue , but we are unable to identify the plugins .

**Reply by Krishnaj20** [reply]
@srikanth-6520 - let us know if you need any specific help from the community on this thread?

**Reply by srikanth-6520** [reply]
@Krishnaj20 , yes , we required the someone help on this issue, kindly help.

**Reply by Krishnaj20** [reply]
@amitpriyadarshi @gandham-santhosh - Can you look into this and provide the necessary assistance?

**Reply by vinukumar-vs** [reply]
@srikanth-6520 Uploading large videos has specific logic of updating the large file in chunks from client side. This logic is specific to Azure(uploading the large file in chunks) today. This implementation change to change if you are using any other CSP provider in File upload editor(Generic editor).

These changes we are doing as part of release-6.0.0(Cloud agnostic). But plugin wont be available for all CSP's. 
@rajeevsathish [@pallakartheekreddy](https://github.com/pallakartheekreddy) @shubhambansaltarento Can you please confirm & share the details if I am missing anything.

@amitpriyadarshi @gandham-santhosh @Krishnaj20

**Reply by chitranshu-keshav** [reply]
@vinukumar-vs Can you please share the JIRA ticket related to this change so that the progress can be tracked and this thread can be marked as answered

cc @ashokreddy1208

**Comment by vinukumar-vs** [comment]
[@pallakartheekreddy](https://github.com/pallakartheekreddy) @shubhambansaltarento
Can you share the details of editor specific changes for this that we have handled as part of release-6.0.0

@srikanth-6520 hope you are verifying this feature with AWS right?

@amitpriyadarshi @gandham-santhosh @Krishnaj20

**Reply by shubhambansaltarento** [reply, marked as answer]
@vinukumar-vs  large video upload in chunks works for azure only
here are the latest release notes for editors from knowledge https://knowlg.sunbird.org/use/release-notes/release-5.5.0

@amitpriyadarshi @gandham-santhosh  @Krishnaj20  [@pallakartheekreddy](https://github.com/pallakartheekreddy)

**Comment by ashokreddy1208** [comment]
@srikanth-6520  is this issue resolved? can you please confirm and update this thread

**Reply by Krishnaj20** [reply]
closing this as we haven't received any response from the requestor.
